### DISCUSSION TITLE
For each enzyme: create and Import an observed data set with just one point: `{simulation_end_time; fraction metabolized}`

### DISCUSSION THREAD

**Discussion by khalid-ahmed** [Open-Systems-Pharmacology/Forum#1747] [Questions & Problems] [answered]
For each enzyme: create and Import an observed data set with just one point: `{simulation_end_time; fraction metabolized}`
Then you can optimize the corresponding Total fraction metabolized of the enzyme vs. this data point

_Originally posted by @Yuri05 in https://github.com/Open-Systems-Pharmacology/Forum/discussions/1476#discussioncomment-5289464_

**Comment by khalid-ahmed** [comment]
Could you please elaborate more on this issue?

**Comment by PavelBal** [comment, marked as answer]
Create an excel file for observed data and create exactly one data point for each enzyme, with time = sufficiently long time value, and value = "metabolization fraction". Then load these data in PK-Sim and use the parameter identification tool to fit your model to these data.

**Comment by SCZ1174** [comment]
Supplement: Simple process (for reference):
- Predicted data: Simulations→Define Settings and Run→Kindey/Organism→Fraction excreted to urine/Total fraction of dose-.
- Observed data: X-axis="Time", Y-axis="Fraction of original form excreted to urine/Total fraction of dose-contributed by metabolizing enzymes".
- Both PK and fraction were fitted in the "Parameter Identifications" module.
The final results are referenced below: 
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/86552551/ad9f4a34-859b-4031-9204-bb8073c6c80d)
